### ISSUE TITLE
DrupalCons/Drupal Camps ?

### ISSUE THREAD

**Issue by klonos** [backdrop/backdrop-issues#1355]
Can we please have a place where we post issues about DrupalCons (dates and locations) and other events so Backdrop people can coordinate and meet there if they plan on attending?

The spark for this issue is the forthcoming [DrupalCampMelbourne 2015](http://dcm2015.drupal.org.au) on November 27-28.

I am working the night of the 27th, but if anyone intents to attend on the 28th (@behindthepage ???), then lets coordinate and meet there :wink: 

I don't know if I would be fit for being a representative there (don't even own a Backdrop t-shirt - shame on me), but if people are wondering, I could point them to the right direction and answer a few questions. ...I promise not to start fights about Backdrop vs. D8 :stuck_out_tongue_winking_eye:

**Comment by quicksketch** [comment]
Hi @klonos, that's great! More representatives at events where Jen and I cannot personally attend are very welcome! Shipping a bunch of swag to Australia might be prohibitive for us, but we should do some research into ways we could get t-shirts or least stickers to you or other volunteers to spread the word about Backdrop.

**Comment by klonos** [comment]
> Shipping a bunch of swag to Australia might be prohibitive for us...

I'm guessing you are calculating shipping cost to AU, right? Just in case it is any easier/cheaper to ship to EU, I am returning to my family in Greece for the holiday season, so will be there from the Dec. 8 till Jan. 4 when I return and could bring back anything you could post there.

I wish I was more of a web developer/coder so I can represent us better (code sprints for example), but I am a computer engineer in profession and merely a Drupal/Backdrop and all-things-web enthusiast.

PS: ...just an idea: we could find someone and print t-shirts (even stickers) here instead of having to ship the whole thing.

**Comment by klonos** [comment]
...to the issue at hand though, can we have an *events* repo or something like that that we could post issues like this one to? Posting it here feels off topic and adds noise in the issue queue that's meant to be for Backdrop code issues.

**Comment by quicksketch** [comment]
> PS: ...just an idea: we could find someone and print t-shirts (even stickers) here instead of having to ship the whole thing.

Yep, printing locally may be a much better option than shipping.

> ...to the issue at hand though, can we have an events repo (or something like that) that we could post issues like this one to? 

I'm not sure if it would help. I don't like the restrictive GitHub permissions (like how no one can label an issue unless they can commit to the repository) and it may not be the best medium. Seems like the more places we have to check the less effective all of them become. That may be the reason you mentioned GitHub instead of some other tool...

Maybe we could use the current Google calendar for Backdrop events and turn it into some kind of social calendar. I thought I saw something similar for D8 release parties, but now I can't find it. :frown:

**Comment by klonos** [comment]
Did not make it to the DrupalCamp after all. Was working on Friday and Saturday was sprint day, so was not motivated enough I guess :disappointed: ...perhaps next year.

We still need a place to coordinate and plan presence in DrupalCamps, so leaving this issue open.
